eng/tox/tox.ini

```diff
@@ -41,6 +41,8 @@ pkgs =
 
 
 [testenv]
+commands_pre=
+    {envbindir}/python -m pip install pip==20.3
```

Comment:
was this change intended in this PR?

Reply:
I don't know why it's popping in. I just rebased from main and submitted, and it's still showing up :|

Reply:
Fixed now.